pkg/util/marshal/legacy/marshal.go

```diff
@@ -1,5 +1,6 @@
 // Package marshal converts internal objects to loghttp model objects.  This package is designed to work with
-//  models in pkg/loghttp/legacy.
+//
+//	models in pkg/loghttp/legacy.
```

Comment:
I think the idea here was to remove the double whitespace

```suggestion
// models in pkg/loghttp/legacy.
```